BUILDING.md

```diff
@@ -28,6 +28,13 @@ Finally, `task` will build the library and serve the reference documentation web
 
 The website should be available on http://localhost:8080 with hot-reloading enabled.
 
+The following example runs the `build` task via Docker.
+This task will rebuild SDK constants.
+
+```
+make task "build -f"
```

Comment:
This should probably go in https://github.com/starfederation/datastar/blob/develop/DOCKER.md.

Reply:
I'll take this change out of this PR for now. Might add as a separate PR.